server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ConnectSessionController.java

```diff
@@ -164,15 +164,15 @@ public ListResponse<CheckResult> check(@PathVariable String sessionId, @RequestB
     }
 
     /**
-     * 对多个数据库进行sql检查 todo 待完善
-     * 
+     * check the contents of multiple sql scripts
+     *
      * @param req
      * @return
      */
     @ApiOperation(value = "sqlCheck", notes = "statically check the contents of multiple sql scripts")
-    @PostMapping("sessions/sqlCheck")
-    public ListResponse<CheckResult> multipleCheck(@RequestBody MultipleSqlCheckReq req) {
-        throw new NotImplementedException("Unsupported now");
+    @PostMapping("/sessions/sqlCheck")
+    public ListResponse<MultipleSqlCheckResult> multipleCheck(@RequestBody MultipleSqlCheckReq req) {
```

Comment:
the concept of 'MultipleSqlCheckResult' is confused, multiple statements or multiple databases?

Reply:
Changing this model class‘s name involves front-end changes, so it is not recommended